kairon/shared/account/processor.py

```diff
@@ -79,7 +80,8 @@ def check_bot_exists(name: str, account: int, raise_exception: bool = True):
         return bot_exists
 
     @staticmethod
-    def add_bot(name: str, account: int, user: str, is_new_account: bool = False, add_default_data: bool = True, **metadata):
+    def add_bot(name: str, account: int, user: str, is_new_account: bool = False, add_default_data: bool = True,
```

Comment:
lets not touch this method.
write a wrapper over it: add_bot_with_template() that calls add_bot() and apply_template()